generator/transit_generator.cpp

```diff
@@ -179,13 +182,28 @@ void CalculateEdgeWeight(vector<Stop> const & stops, vector<transit::Edge> & edg
     e.SetWeight(lengthInMeters / kTransitAverageSpeedMPS);
   }
 }
+
+void FillOsmIdToFeatureIdMap(string const & osmIdsToFeatureIdPath, OsmIdToFeatureIdsMap & map)
+{
+  CHECK(ForEachOsmId2FeatureId(osmIdsToFeatureIdPath,
+                               [&map](osm::Id const & osmId, uint32_t featureId) {
+                                 map[osmId].push_back(featureId);
+                               }),
+        ());
```

Comment:
It would be nice to print at least path in the diagnostic message.

Reply:
Done